data_access.py

```diff
@@ -73,4 +75,12 @@ def decide_feature_name(feature_path):
             feature_2[f_2] = feature[f]
         else:
             feature_2[f] = feature[f]
-    return feature_2
\ No newline at end of file
+    return feature_2
+
+
+def callfunc(my_file):
+    pathname, filename = os.path.split(my_file)
+    sys.path.append(os.path.abspath(pathname))
+    modname = os.path.splitext(filename)[0]
+    my_mod = importlib.import_module(modname)
+    return my_mod
```

Comment:
This collects the .py file for the model you show the framework on learning_models.json
this function finds the model of .py file its functions and classes to run on model_processor.py